product_docs/docs/biganimal/release/pricing_and_billing/index.mdx

```diff
@@ -22,8 +22,16 @@ Extreme high availability powered by EDB Postgres Distributed is now available i
 
 ## Cloud infrastructure costs
 
+### BigAnimal's cloud account
+
 If you're connecting directly to BigAnimal through BigAnimal's cloud account, EDB bills you for cloud infrastructure, such as compute, storage, data transfer, monitoring, and logging. You can view monthly estimates for BigAnimal's cloud infrastructure costs while you're creating a cluster. 
```

Comment:
change to: 
"If you're connecting directly to BigAnimal through BigAnimal's cloud account, EDB bills you for cloud infrastructure. You can view monthly estimates for BigAnimal's cloud infrastructure costs while you're creating a cluster."